test/config-serializer.spec.js

```diff
@@ -29,9 +50,40 @@ describe('Config Serializer', () => {
     };
 
     let str = serializeConfig(cfg);
-    expect(str).to.be.equal(outCfg);
+    expect(str).toBe(outCfg);
+  });
+
+  it('can serialize System configuration', () => {
```

Comment:
The "format" is actually same for both `System` and `SystemJS`. The recent version of jspm uses `SystemJS` global object which is basically an alias for `System` global object.